tests/templates/kuttl/mount-dags-gitsync/31-assert.yaml.j2

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+metadata:
+  name: metrics
+timeout: 30
+commands:
+
+{% if test_scenario['values']['executor'] == 'kubernetes' %}
+  # check that the executor template configmap contains mounts and envs
+  # will expect 4 (2 from from the volume declaration + mounts to two containers, base and gitsync)
+  - script: kubectl -n $NAMESPACE get cm airflow-executor-pod-template -o json | jq -r '.data."airflow_executor_pod_template.yaml"'  | grep "test-cm-gitsync" | wc -l | grep 4
```

Comment:
I know we want this merged fast but probably a more reliably test would be to check for the existence of these things (CMs, env vars) directly on the airflow pods. Maybe even write a small DAG that does just this.

Reply:
The difficulty here - for the kubernetesExecutor case - is that the templated pods disappear quickly, so I thought it more reliable to check the configMap that is always "there". I could maybe add a statefulset check for the celeryExecutor case, but wanted to keep it consistent.

Reply:
Thanks for the explnation